docs/GettingStarted.md

```diff
@@ -207,7 +207,7 @@ Available settings are:
 * ``enabled``: define if the ``tracer`` is enabled or not. If set to
   ``false``, the code is still instrumented but no spans are sent to the local
   trace agent.
-* ``default_service``: set the service name used when tracing application
+* ``sidekiq_service``: set the service name used when tracing application
```

Comment:
Why not just `service`? (I tried to compare to what we have in Python, there we still have a mix of "default_service" and "service" for middleware configuration).